utilities/tools/MARManifestGenerator/manifestHeader.yml

```diff
@@ -0,0 +1,330 @@
+version: 2.0
+subscriptionId: 22ec7257-9128-45a4-9751-69ef4f22a75b
+resourceGroup: bicep-public-registry-rg
+registry: biceppublic
+onboardingNotification: brmmcronboarding
+dependencyNotification: brmmcrdependency
+
+ownership:
+  securityGroups:
+    - groupName: redmond\bicepmcr
```

Comment:
Overall I got to wonder if any of these variables and names open an attack surface.

Could you advice, @matebarabas?